src/templates/layout/base.twig

```diff
@@ -27,6 +27,20 @@
                         <li><a href="{{ url('watch.list') }}">Watch Functions</a></li>
                         <li><a href="{{ url('waterfall.list') }}">Waterfall</a></li>
                     </ul>
+                    {% set sites = sites() %}
```

Comment:
This should probably be passed into the template context as part of the template being rendered. That would help solve the issue with globals as well.